### ISSUE TITLE
Next Step Microgrant: Buy me a Coffee

### ISSUE THREAD

**Issue by dev-blockchain** [filecoin-project/devgrants#358]
### 1. What is your project? (max 100 words)

Buy me a Coffee is a platform to ask fans to support your work for the price of a coffee. In exchange, they receive a unique minted NFT and exclusive access to a private community hosted on the blockchain.

https://github.com/dev-blockchain/buy-me-a-coffee

### 2. How are you planning to improve this project? (max 200 words)
Adding the following features to the application:
- Creators can create a private community for their donors and fans
- Integrating IPFS and Filecoin storage so the creators can share exclusive content with their supporters
- Donors can vote on the upcoming work of their favorite creators and share their ideas with them
- An attractive UI friendly badge that can be added by creators to their Github, Youtube, Instagram, etc. to request the support
 
### 3. Will the work be Open Source?
This project is open source and is using MIT license

### 4. If selected, do you agree to share weekly updates and a grant report upon conclusion?
Yes
  
### 5. Does your proposal comply with our Community Code of Conduct?
Yes

### 6. Links and submissions

  
* Have you submitted to the Filecoin Community Showcase ([instructions](https://github.com/filecoin-project/community#showcase-your-project))?
  
Yes, created the pull request. Here is the link: https://github.com/filecoin-project/community/pull/348

* Have you created a thread introducing your project in the community discussion board's [ Project Showcase category](https://github.com/filecoin-project/community/discussions/categories/project-showcase)?
  
  
Yes, created the thread. Here is the link: https://github.com/filecoin-project/community/discussions/347 

* If your project began at a hackathon, have you submitted it for the relevant Protocol Labs prizes? Include links here if available:
### Additional questions:
* For each team member(s), please list name, email, Github account, and role in the project.
* [Deepesh - Developer]; Github Account: @dev-blockchain; [Email](graphical.eye@gmail.com)
* If your project was created as part of an event or hackathon:
  * What was the name of the event? (e.g. ETHGlobal NFTHack, Cal Hacks hello:world, Chainlink, CivHacks, GameDevJ, ETHGlobal Scaling Ethereum)
  * Please link to your hackathon submission
  * Accepted in Gitcoin Grants Round 11 Microgrants: [Hackathon: Grants Round 11 Hackathon [Gitcoin] Microgrants](https://gitcoin.co/issue/filecoin-project/devgrants/326/100026624)

**Comment by ErinOCon** [comment]
@dev-blockchain, my sincere apologies for the significant delay. If you are still interested in receiving funding, can you update your Github link? The one provided does not appear to be active.

**Comment by ErinOCon** [comment]
@dev-blockchain, just circling back to confirm if you are still interested in submitting this proposal. Thanks!

**Comment by dev-blockchain** [comment]
@ErinOCon Thank you for getting back.

I have updated the Github repository, and it should be working now.

Please let me know if anything else is needed.

**Comment by ErinOCon** [comment]
Hi @dev-blockchain, Thank you for the update! Have you received a microgrant to support this work previously?

**Comment by dev-blockchain** [comment]
@ErinOCon No, I did not receive a microgrant previously to support this work. 

I participated in the Filecoin Hackathon on Gitcoin, and the team shared the instructions to submit the project in this Github thread to apply for the microgrant.

**Comment by ErinOCon** [comment]
Thank you for confirming, @dev-blockchain!

**Comment by ErinOCon** [comment]
Hi @dev-blockchain, Thank you for all of your patience. This microgrant has been approved! Can you provide a current email address to proceed with the payment process?

**Comment by dev-blockchain** [comment]
@ErinOCon Thank you! 

My email address is: graphical.eye@gmail.com
